input/kafkamdm/kafkamdm.go

```diff
@@ -46,7 +49,7 @@ var topics []string
 var partitionStr string
 var partitions []int32
 var offsetStr string
-var dataDir string
+var DataDir string
```

Comment:
mt-kafka-mdm-sniff overrides it. it deliberately needs to use a different datadir after the configuration is loaded, otherwise it would conflict with metrictank's database.